src/main/java/baritone/process/elytra/BlockStateOctreeInterface.java

```diff
@@ -39,16 +39,16 @@ public BlockStateOctreeInterface(final NetherPathfinderContext context) {
     }
 
     public boolean get0(final int x, final int y, final int z) {
-        if ((y | (127 - y)) < 0) {
+        if (y < 0 || y > 383) {
```

Comment:
This always checks for the maximum chunk height, is that correct? i.e. does nether-pathfinder always allocate full-size chunks?

Reply:
yes